demo/redirect-flow-example/src/App.tsx

```diff
@@ -398,7 +382,36 @@ function App() {
       uiConsole(sig.toString("hex"));
     }
   };
+  const signMessageWithPrecomputedTss = async (): Promise<any> => {
+    if (coreKitInstance.keyType === "secp256k1") {
+      const precomputedTssClient = await coreKitInstance.precompute_secp256k1();
+      const msg = Buffer.from("hello signer!");
+      const sig = await coreKitInstance.sign(msg, false, precomputedTssClient);
+      uiConsole(sig.toString("hex"));
+    } else if (coreKitInstance.keyType === "ed25519") {
+      const msg = Buffer.from("hello signer!");
+      const sig = await coreKitInstance.sign(msg);
+      uiConsole(sig.toString("hex"));
+    }
+  };
+
+  const signMultipleMessagesWithPrecomputedTss = async (): Promise<any> => {
+    if (coreKitInstance.keyType === "secp256k1") {
+      const precomputedTssClient = await coreKitInstance.precompute_secp256k1();
+      const precomputedTssClient2 = await coreKitInstance.precompute_secp256k1();
+
+      const msg = Buffer.from("hello signer!");
+      const sig = await coreKitInstance.sign(msg, false, precomputedTssClient);
```

Comment:
should use Promis.all to showcase parallel for precompute and sign

Reply:
done

Reply:
`Promise.all()` will not make it parallel. There is distinct difference between concurrent and parallel computation.